src/polyfill/string.js

```diff
@@ -23,6 +23,7 @@ defineProtoFunc(String, 'includes', function(search, start) {
 });
 
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith#Polyfill
-defineProtoFunc(String, 'startsWith', function(search, pos) {
-    return this.substr(!pos || pos < 0 ? 0 : +pos, search.length) === search;
+defineProtoFunc(String, 'startsWith', function(search, rawPos) {
+    var pos = rawPos > 0 ? rawPos|0 : 0;
+    return this.substring(pos, pos + search.length) === search;
```

Comment:
```suggestion
defineProtoFunc(String, 'startsWith', function(search, pos) {
    pos = rawPos > 0 ? rawPos|0 : 0;
    return this.substring(pos, pos + search.length) === search;
```

(removing `var` while reusing `pos` variable)

Reply:
I know this code can be tweaked but I'm just updating what's here to the linked Polyfill on MDN. I'd rather keep this in sync with that.